src/main/java/backend/UpdateController.java

```diff
@@ -87,7 +89,13 @@ private void openRepositoriesInFilters(List<FilterExpression> filterExprs) {
         return filterExprs.stream()
                 .filter(Qualifier::hasUpdatedQualifier)
                 .flatMap(filterExpr -> allModelIssues.stream()
-                        .filter(issue -> Qualifier.process(models, filterExpr, issue)))
+                        .filter(issue -> {
+                            try {
+                                return Qualifier.process(models, filterExpr, issue);
+                            } catch (SemanticException e) {
+                                return false;
+                            }
+                        }))
```

Comment:
Yes, I'm ok with not distinguishing between errors and warnings.
If @raisfathin can finish the repo picker soon, we can use the area currently occupied by the repo dropdown to show feedback/errors/warnings (as we discussed last week). The status bar below can be used to show status of automatic actions such as refreshes.